ingest/clusters.py

```diff
@@ -46,30 +48,30 @@ def transform(self, row):
 
```

Comment:
I think the above needs to actually be this:

```
annotation = self.header[idx]
if annotation.casefold() in ('name', 'x', 'y', 'z'):
    annotation = annotation.casefold()
```
Otherwise we get a KeyError when trying to look up the annotation.  There's another fix that needs to happen on line 73.

Reply:
Addressed in 987cc80